codeceptjs-cucumber/codecept.conf.js

```diff
@@ -38,7 +38,7 @@ exports.config = {
     testomat: {
       enabled: true,
       require: '@testomatio/reporter/lib/adapter/codecept',
-      apiKey: 'bu8e29984sqd',
+      apiKey: 'w2n70ad83nzq',
```

Comment:
env variable!


also please add to readme notification that API key should not be stored in codebase but saved in .env file or as environment var